app/src/containers/b2b/AccountMain.less

```diff
@@ -3,148 +3,96 @@
 .account-component {
   height: 100%;
   display: flex;
-  flex-direction: column;
-
-  .account-header {
-    flex: 0 0 auto;
-    padding: 20px 0px 10px;
+  flex-wrap: wrap;
+  padding-bottom: 50px;
 
-    @media (max-width: @tabletWidth - 1px) {
-      display: flex;
-      box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.3);
-      padding: 16px 15px;
+  .account-tree-section {
+    width: 250px;
+    flex: 1 1 250px;
+    margin-top: 20px;
+    margin-right: 30px;
+    @media (max-width: 859px) {
+      margin-right: 0;
+    }
+    @media (max-width: @mobileWidth - 1px) {
```

Comment:
Let's quickly confirm with @BonnieEP if we need to hide this in mobile.

Reply:
@petro97 Can you please take a look at the AM-UI and follow the same behaviour for displaying the sub-accounts in mobile as that? Will follow up on slack.